File: src/test/java/ukalus/items/ArtDescriptionResourceTest.java
/*
 * Created on Feb 25, 2005
 *
 */
package ukalus.items;

import static java.lang.System.out;

import java.util.ResourceBundle;

import junit.framework.TestCase;

/**
 * @author gremopm
 * 
 */
public class ArtDescriptionResourceTest extends TestCase {

  public void testGenerate() throws Exception {
    out.println(String.valueOf(ResourceBundle.getBundle(ArtDescriptionResource.class.getName())));
  }

}

File: src/test/java/ukalus/items/ArtDescriptionResourceTest.kt
/*
 * Created on Feb 25, 2005
 *
 */
package ukalus.items

import junit.framework.TestCase
import java.util.*

/**
 * @author gremopm
 */
class ArtDescriptionResourceTest : TestCase() {

    fun testGenerate() {
        println(ResourceBundle.getBundle(ArtDescriptionResource::class.java.name).inspect())
    }

}

fun ResourceBundle.inspect(): String {
    return keys
            .asSequence()
            .joinToString(separator = "\n", transform = { it -> "$it=${getString(it)}" })
}